project-1/main.py

```diff
@@ -0,0 +1,57 @@
+import gspread
```

Comment:
Refactoring this into a functional script will make things cleaner, and docstring to the function will help

Reply:
Good afternoon sir, please i would like more elaboration on this i have made my research but it is leading me nowhere.

Reply:
Refactor your code, break down the parts that perform a single task into functions

Reply:
Hi @elseagle i have done all the functional scripts i would like a feedback about it